client/components/Forms/EventForm.js

```diff
@@ -106,41 +129,84 @@ class EventForm extends Component {
     const { formData } = this.state;
     const newFormData = Object.assign({}, formData);
 
-    newFormData[e.target.name] = e.target.value.trim();
+    newFormData[e.target.name] = e.target.value;
```

Comment:
Just curiosity, why are we removing the trim?

Reply:
It would be better if we have the `trim()` not removed though, is there a reason... 

For context : https://www.w3schools.com/jsref/jsref_trim_string.asp

Reply:
I think the trim should be done when the form is going to be submitted. The reason I removed it here is that the change listener doesn't handle `spaces` well when the trim is where it was.
@andela-sjames @chibujax

Reply:
Gotcha